firmware/boron-ins-fsm/src/stateMachine.cpp

```diff
@@ -35,298 +37,259 @@ std::queue<int> reasonQueue;
-    stateHandler = state2_duration;
-  }
-  else if(millis() - state3_stillness_timer >= state3_max_stillness_time){
-
-    Log.warn("stillness alert, remaining in state3 after publish");
-    saveStateChangeOrAlert(3, 5);
-    Log.error("Stillness Alert!!");
-    Particle.publish("Stillness Alert", "stillness alert!!!", PRIVATE);
-    state3_stillness_timer = millis();  // reset the stillness timer
-    stateHandler = state3_stillness;
-  }
-  else {
-    //if we don't meet the exit conditions above, we remain here
-    //stateHandler = state3_stillness;
-
-  }
-
-} //end state3_stillness
-
-void publishStateTransition(int prevState, int nextState, unsigned char doorStatus, float INSValue){
-
-    if(stateMachineDebugFlag){
-
-      //from particle docs, max length of publish is 622 chars, I am assuming this includes null char
-      char stateTransition[622];
-      snprintf(stateTransition, sizeof(stateTransition), 
-              "{\"prev_state\":\"%d\", \"next_state\":\"%d\", \"door_status\":\"0x%02X\", \"INS_val\":\"%f\"}", prevState, nextState, doorStatus, INSValue); 
-      Particle.publish("State Transition", stateTransition, PRIVATE);
+}  // end state3_stillness
 
+void publishStateTransition(int prevState, int nextState, unsigned char doorStatus, float INSValue) {
+    if (stateMachineDebugFlag) {
+        // from particle docs, max length of publish is 622 chars, I am assuming this includes null char
+        char stateTransition[622];
+        snprintf(stateTransition, sizeof(stateTransition),
+                 "{\"prev_state\":\"%d\", \"next_state\":\"%d\", \"door_status\":\"0x%02X\", \"INS_val\":\"%f\"}", prevState, nextState, doorStatus, INSValue);
+        Particle.publish("State Transition", stateTransition, PRIVATE);
     }
-
 }
 
-void publishDebugMessage(int state, unsigned char doorStatus, float INSValue, unsigned long timer){
-
+void publishDebugMessage(int state, unsigned char doorStatus, float INSValue, unsigned long timer) {
     static unsigned long lastDebugPublish = 0;
 
-    if(stateMachineDebugFlag && (millis()-lastDebugPublish) > DEBUG_PUBLISH_INTERVAL){
-
-      //from particle docs, max length of publish is 622 chars, I am assuming this includes null char
-      char debugMessage[622];
-      snprintf(debugMessage, sizeof(debugMessage), 
-              "{\"state\":\"%d\", \"door_status\":\"0x%02X\", \"INS_val\":\"%f\", \"timer_status\":\"%ld\"}", state, doorStatus, INSValue, timer); 
-      Particle.publish("Debug Message", debugMessage, PRIVATE);
-      lastDebugPublish = millis();
-
+    if (stateMachineDebugFlag && (millis() - lastDebugPublish) > DEBUG_PUBLISH_INTERVAL) {
+        // from particle docs, max length of publish is 622 chars, I am assuming this includes null char
+        char debugMessage[622];
+        snprintf(debugMessage, sizeof(debugMessage),
+                 "{\"state\":\"%d\", \"door_status\":\"0x%02X\", \"INS_val\":\"%f\", \"timer_status\":\"%lu\"}", state, doorStatus, INSValue, timer);
```

Comment:
Why did the last formatting change from `%ld` to `%lu`?

Reply:
This was to fix a complaint from cppcheck saying that the correct syntax to print timer, an unsigned long, is not %ld